realip_test.go

```diff
@@ -75,10 +76,14 @@ func TestRealIP(t *testing.T) {
 			name:     "Has X-Forwarded-For",
 			request:  newRequest("", "", publicAddr1),
 			expected: publicAddr1,
+		}, {
+			name:     "Has X-Forwarded-For multiple IPs",
+			request:  newRequest("", "", fmt.Sprintf("%s,%s", localAddr, publicAddr1)),
```

Comment:
The [wiki article](https://en.wikipedia.org/wiki/X-Forwarded-For) claims the common pattern is spaces after the comma. Should we test that, as well?

Reply:
Yes. I will add a test case for the space. thanks.